setup.py

```diff
@@ -159,8 +160,9 @@ def run(self):
             "heroku = sentry_plugins.heroku.plugin:HerokuPlugin",
             "jira = sentry_plugins.jira.plugin:JiraPlugin",
             "jira_ac = sentry_plugins.jira_ac.plugin:JiraACPlugin",
-            "phabricator = sentry_plugins.phabricator.plugin:PhabricatorPlugin",
+            "opsgenie = sentry_plugins.opsgenie.plugin:OpsGeniePlugin",
             "pagerduty = sentry_plugins.pagerduty.plugin:PagerDutyPlugin",
+            "phabricator = sentry_plugins.phabricator.plugin:PhabricatorPlugin",
```

Comment:
I realized that the `phabricator` should be after `pagerduty` so I made that change as well